docs/solutions/hypervisors/kvm/kvm-baremetal/main.tf

```diff
@@ -0,0 +1,74 @@
+provider "oci" {
+  tenancy_ocid     = "${var.tenancy_ocid}"
+  user_ocid        = "${var.user_ocid}"
+  fingerprint      = "${var.fingerprint}"
+  private_key_path = "${var.ssh_private_key_path}"
```

Comment:
The ssh key should be different from the key used to make requests. (See the "important" message at https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/apisigningkey.htm.) Should have a separate variable for private_key_path.

Reply:
This private key is used on the template to perform `remote-exec` operations (SSH/SCP) and used within a [connection](https://www.terraform.io/docs/provisioners/connection.html) block.  So, per my understanding, this should be the RSA key, not the PEM format used by the SDK. I just need remove it from the provider declaration and perhaps provider a better name to differentiate from the `private_key_path` variable.

Reply:
Correct - see for example https://github.com/oracle/terraform-provider-oci/blob/master/docs/solutions/omc_monitored_server/env-vars